src/index.html

```diff
@@ -7,6 +7,39 @@
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
   <body>
-    <h1>Calendar</h1>
+    <div class="calendar calendar--month-length">
+      <div class="calendar__day calendar__day--start-day_sun"></div>
+      <div class="calendar__day calendar__day-1"></div>
```

Comment:
Don’t need to add 1, 2, 3 for each day. Use nth-child

```suggestion
      <div class="calendar__day"></div>
```